tests/test_html_text.py

```diff
@@ -149,12 +163,24 @@ def test_personalize_newlines_sets():
     assert text == 'text\n\nmore\n\nand more text\n\nand some more'
 
 
-def test_webpages():
-    webpages = sorted(glob.glob('./test_webpages/*.html'))
-    extracted = sorted(glob.glob('./test_webpages/*.txt'))
-    for page, extr in zip(webpages, extracted):
-        with open(page, 'r', encoding='utf8') as f_in:
-            html = f_in.read()
-        with open(extr, 'r', encoding='utf8') as f_in:
-            expected = f_in.read()
-        assert extract_text(html) == expected
+def _webpage_paths():
+    webpages = sorted(glob.glob(os.path.join(ROOT, 'test_webpages', '*.html')))
+    extracted = sorted(glob.glob(os.path.join(ROOT, 'test_webpages','*.txt')))
+    return list(zip(webpages, extracted))
+
+
+def _load_file(path):
+    with open(path, 'rb') as f:
+        return f.read().decode('utf8')
+
+
+@pytest.mark.parametrize(['page', 'extracted'], _webpage_paths())
+def test_foo(page, extracted):
```

Comment:
Probably ``foo`` was meant as a temporary placeholder for
```suggestion
def test_webpages(page, extracted):
```